packages/rulesets/src/spectral/az-arm.ts

```diff
@@ -988,6 +988,8 @@ const ruleset: any = {
       description: "This rule checks for references that aren't using latest version of common-types.",
       message: "{{error}}",
       severity: "warn",
+      disableForTypeSpec: true,
+      disableForTypeSpecReason: "TODO",
```

Comment:
@tjprescott @timotheeguerin  what should be the explanation for disabling `LatestVersionOfCommonTypesMustBeUsed` ?

Reply:
We should not disable it.